tests/scale/test_big_bundles.py

```diff
@@ -6,6 +6,8 @@
 
 
 class TestBigBundles(unittest.TestCase):
+    # deployment = os.environ['CI_COMMIT_REF_NAME']
+    deployment = "integration"
```

Comment:
are you going to change this back?

Reply:
It seems like better practice to set the deployment once and then reference the internal declaration than to call the environment variable twice in the dataset fixture/runner declarations. 

I'm especially partial to this because every feature branch I push requires the deployment hack, so I'm changing these types of lines frequently.

Reply:
I think the intention is correct. We shouldn't need to hack the environment in a PR branch. Maybe we can have logic in the top-level makefile that sets a `DCP_DEPLOYMENT` environment variable use that variable here. The variable should set to `integration` on PR branches and to CI_COMMIT_REF_NAME otherwise. Sth like that.

However, I think we shouldn't mix these two concerns in one PR and this hack should be reverted prior to landing this PR.